Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -1078,6 +1078,15 @@ private static void InvokeRpcRegisterCallBack(string dataJson, string senderJson
                 Debug.LogError(ex.Message);
             }
 
+
```

Comment:
A bit of description Why this needs to be done would be great, somewhere in the code. So the future you can send blessings on the current you.

Reply:
@momintlh Could you add a description? This implementation could easily be forgotten in the future.